sfm/ui/jobs.py

```diff
@@ -60,21 +63,29 @@ def seedset_harvest(seedset_id):
     for historical_seed in historical_seeds:
         if historical_seed.is_active:
             seed_map = {}
+            seed_map["seed_id"] = historical_seed.seed_id
             if historical_seed.token:
                 seed_map["token"] = historical_seed.token
             if historical_seed.uid:
                 seed_map["uid"] = historical_seed.uid
             message["seeds"].append(seed_map)
 
-    routing_key = "harvest.start.{}.{}".format(historical_credential.platform, harvest_type)
+    # Include conditions to check Is Active Flag here.
```

Comment:
Is adding this necessary given that there is an is active check at the beginning of this function?